### PR TITLE
Sound radius is min(volume * 16, 16)

### PR DESCRIPTION
**Pull request by coelho**
The sound radius used in vanillia is min(volume \* 16, 16)

### CODE REVIEW

File: src/main/java/net/glowstone/GlowWorld.java [hunk 1]
```diff
@@ -1072,7 +1072,7 @@ public void playEffectExceptTo(Location location, Effect effect, int data, int r
     public void playSound(Location location, Sound sound, float volume, float pitch) {
         if (location == null || sound == null) return;
 
-        final int radiusSquared = 24 * 24; // todo: verify this radius
+        double radiusSquared = Math.pow(Math.min(volume * 16D, 16D), 2);
```
Comment (turt2live) [inline src/main/java/net/glowstone/GlowWorld.java]: Don't need the `D`'s

**Comment by turt2live** [comment]
@coelho this PR does not currently merge. Please fix that ASAP!

**Comment by coelho** [comment]
@turt2live Should be fixed.

**Comment by turt2live** [comment]
Looks good to go

**Comment by SpaceManiac** [comment]
Merged in a4d17e2f08cb81def12a5d806d30a09185411b47.

**Pull request closed without merging**